src/privatekey.hpp

```diff
@@ -34,6 +34,9 @@ class PrivateKey {
     // Construct a private key from a BIP32 based seed.
     static PrivateKey FromSeedBIP32(const Bytes& seed);
 
+    // Construct a random private key.
```

Comment:
Publicly-exposed calls like these should not be exposed as part of `PrivateKey`, proper. It might be better to make this a function within `test.cpp` to ensure that it isn't used anywhere outside that.

Reply:
The point here, is that external libraries actually might want to be able to make a random private key.